src/test/java/redis/clients/jedis/ClusterPipeliningTest.java

```diff
@@ -0,0 +1,883 @@
+
+        p.sync();
+        Assert.assertEquals(Long.valueOf(4), r1.get());
+        Assert.assertEquals(Long.valueOf(4), r2.get());
+        Assert.assertEquals(Long.valueOf(0), r3.get());
+        Assert.assertEquals(Long.valueOf(0), r4.get());
+        Assert.assertEquals(1, r5.get().size());
+        Assert.assertEquals("OK", r6.get());
+        Assert.assertEquals(Long.valueOf(2), r7.get());
+        Assert.assertEquals("hello", r8.get());
+        Assert.assertEquals("OK", r9.get());
+        Assert.assertEquals(Long.valueOf(1), r10.get());
+        Assert.assertEquals(vals, r11.get());
+        Assert.assertEquals("foo", r12.get());
+        Assert.assertEquals(vals, r13.get());
+        Assert.assertEquals(2, r14.get().size());
+        Assert.assertEquals(Long.valueOf(2), r15.get());
+        Assert.assertEquals(Long.valueOf(0), r16.get());
+        Assert.assertEquals(Long.valueOf(0), r17.get());
+        Assert.assertEquals("world", r18.get());
+        Assert.assertEquals("hello", r19.get());
+    }
+
+    @Test
+    public void clusterPipelineSet() {
+        Set<String> diff = new HashSet<>();
+        diff.add("bar");
+        diff.add("foo");
+
+        Set<String> union = new HashSet<>();
+        union.add("hello");
+        union.add("world");
+        union.add("bar");
+        union.add("foo");
+
+        Set<String> inter = new HashSet<>();
+        inter.add("world");
+        inter.add("hello");
+
+        ClusterConnectionProvider provider = new ClusterConnectionProvider(nodes, DEFAULT_CLIENT_CONFIG);
+        ClusterPipeline p = new ClusterPipeline(provider);
+
+        Response<Long> r1 = p.sadd("myset{|}", "hello", "hello", "world", "foo", "bar");
+        p.sadd("mynewset{|}", "hello", "hello", "world");
+        Response<Set<String>> r2 = p.sdiff("myset{|}", "mynewset{|}");
+        Response<Long> r3 = p.sdiffstore("diffset{|}","myset{|}", "mynewset{|}");
+        Response<Set<String>> r4 = p.smembers("diffset{|}");
+        Response<Set<String>> r5 = p.sinter("myset{|}", "mynewset{|}");
+        Response<Long> r6 = p.sinterstore("interset{|}","myset{|}", "mynewset{|}");
+        Response<Set<String>> r7 = p.smembers("interset{|}");
+        Response<Set<String>> r8 = p.sunion("myset{|}", "mynewset{|}");
+        Response<Long> r9 = p.sunionstore("unionset{|}","myset{|}", "mynewset{|}");
+        Response<Set<String>> r10 = p.smembers("unionset{|}");
+        Response<Boolean> r11 = p.sismember("myset{|}", "foo");
+        Response<List<Boolean>> r12 = p.smismember("myset{|}", "foo", "foobar");
+        Response<Long> r13 = p.srem("myset{|}", "foo");
+        Response<Set<String>> r14 = p.spop("myset{|}", 1);
+        Response<Long> r15 = p.scard("myset{|}");
+        Response<String> r16 = p.srandmember("myset{|}");
+        Response<List<String>> r17 = p.srandmember("myset{|}", 2);
+        Response<Long> r18 = p.smove("myset{|}", "mynewset{|}", "hello");
```

Comment:
Could you please use a better hashtag? E.g. `set`, as in `my{set}`, `mynew{set}`, etc.